### ISSUE TITLE
Updating email address isn't working

### ISSUE THREAD

**Issue by nicholastic** [brandonseydel/MailChimp.Net#201]
I'm trying to update a subscribed member's email address using AddOrUpdateAsync. I'm retrieving the member first. Then I create a new member, setting his ID to the ID of the old member, and his email address to the new email address. It's not working, and I'm getting " http://developer.mailchimp.com/documentation/mailchimp/guides/error-glossary/; Title: Invalid Resource; Status 400; Detail: The resource submitted could not be validated. For field-specific details, see the 'errors' array."

Am I doing this right? Previously I was deleting and recreating. Here, I set Status first to subscribed, which is what he was in the first place. Then I tried not setting Status at all. I haven't set StatusIfNew because I'm only updating an existing member. What am I doing wrong?

Actually, I think this isn't working because the error message is: "\"john@example.com\" is already in this list with a status of \"subscribed\"." Which is crazy, since I deleted him and he's not in the list at all. Found this: http://stackoverflow.com/questions/36494975/can-not-re-subscribe-deleted-member

Sorry, just wanted to confirm that it is working. It's Mailchimp's fault, they won't let you change the email address if the new email address belongs to a member who has been deleted from the list. They don't actually remove it on their end even though it's invisible from the user interface, the playground, and curl.

So basically I set the ID of the new member object to the ID of the old member object and filled in the new email address, setting the Status of the new member to the Status of the old member. Want me to write some documentation for how to change the email address? It wasn't immediately intuitive.
